.buildkite/pipeline.yml

```diff
@@ -57,7 +42,12 @@ steps:
       - label: ":go: helm-tests"
         command: "make helm-test"
 
-  # Different e2e tests are run depending on the branch and the build source
+  # -- Build --
+
+  - label: ":buildkite:"
+    command: buildkite-agent pipeline upload .buildkite/pipeline-build.yml
```

Comment:
Do we also want to just go ahead and move our `lint`, `checks`, `e2e-tests` groups into their own pipeline file, similar to this to simplify our primary `pipeline.yml` file?

Reply:
I was hesitant to do this and left it as I find the steps in the `checks` and `tests` groups have simple command and are therefore very readable. I found the indirection not interesting here.

Reply:
This was primarily a suggestion and certainly not required. 👍